skymp5-server/cpp/server_guest_lib/MpChangeForms.h

```diff
@@ -105,6 +106,9 @@ class MpChangeFormREFR
   // Used for SetDisplayName (object reference)
   std::optional<std::string> displayName;
 
+  // Used for Quest (QUST) synchronization
+  std::optional<std::vector<Quest>> quests;
```

Comment:
vector<optional<Quest>> -> optional<vector<Quest>>

Reply:
110 seems like you wrote to change optional<vector>, what is wrong?

Reply:
std::optional<std::vector>

Reply:
sorry